### PR TITLE
Math Magicians: Refactor with Hooks

### PR DESCRIPTION
**Pull request by yigitm**
Below actions are done;

- Calculator.js is changed into a function component
- App.js is changed into a function component
- Calculator HTML display is refactored

Thank you for your time.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,106 +1,82 @@
...
+  const [initial, setInitial] = useState(0);
+
+  function getData(e) {
+    setObj(calculate(obj, e.target.textContent));
+    setInitial('');
+  }
+
+  function allClear() {
+    setInitial('0');
   }
```
Comment (DeliceLydia) [inline src/components/Calculator.js:20]: - [ ] I suggest that you avoid using `function` keyword and you use the `arrow function` to practice ES6 features, and it arrows you to create a function in a cleaner way compared to regural function

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,106 +1,82 @@
...
+    '6',
+    '-',
+    '1',
+    '2',
+    '3',
+    '+',
+    '0',
+    '.',
+    '=',
+  ];
```
Comment (DeliceLydia) [inline src/components/Calculator.js:41]: - There is something I realized when performing an operation like `9+2` on your screen it is `2+9`, operator `+` should come after the first number please fix it
op

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @yigitm, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! major highlights for your project:
- [ ] No linter errors
- [ ] Good PR


## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (yigitm) [reply src/components/Calculator.js:41]: Thank you. I changed the output placement.

File: src/components/Calculator.js (hunk 1 above)
Comment (yigitm) [reply src/components/Calculator.js]: Thank you. I changed it to ES6 syntax.

**Review by ZeenatLawal — APPROVED** [review]
Hi @yigitm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**